oeps/oep-0014.rst

```diff
@@ -0,0 +1,175 @@
+=========================================
+OEP-14: Archiving edX Github Repositories
+=========================================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-14 </oeps/oep-0014.rst>`        |
++---------------+-------------------------------------------+
+| Title         | Archiving edX Github Repositories         |
++---------------+-------------------------------------------+
+| Last Modified | 2017-01-09                                |
++---------------+-------------------------------------------+
+| Author        | Christina Roberts, christina@edx.org      |
++---------------+-------------------------------------------+
+| Arbiter       | TBD                                       |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Process                                   |
++---------------+-------------------------------------------+
+| Created       | 2017-01-09                                |
++---------------+-------------------------------------------+
+| `References`  | `ORA PR Discussion`_,                     |
+|               | `Initial Archiving Discussions`_          |
++---------------+-------------------------------------------+
+
+.. _ORA PR Discussion: https://github.com/edx/edx-ora/pull/187
+.. _Initial Archiving Discussions: https://openedx.atlassian.net/wiki/display/IT/Proposed+Github+Deprecation+Process
+
+Abstract
+========
+
+The `edx organization`_ contains a large number of repositories, most of
+which are active and maintained, but some of which are now obsolete. To
+clarify the status of repositories, and to ensure that we are only monitoring
+active repositories via Gemnasium (which alerts about the usage of third
+party libraries with known security issues), a process for archiving a
```

Comment:
Instead of only calling out Gemnasium - can we just refer to "GitHub Organization-wide tools" and list Gemnasium as an example?  We use VersionEye on other repos, and may switch.  I'm sure there are other tools that in the future we'd let consume all of the repos in edx due to testing/compliance/etc.